earth2studio/models/px/aifs.py

```diff
@@ -716,17 +586,15 @@ def _prepare_input(
 
         return x
 
-    def _copy_output_to_input(
+    def _update_input(
         self,
         x: torch.Tensor,
-        y: torch.Tensor,
         coords: CoordSystem,
     ) -> torch.Tensor:
-        """Copy output tensor to input tensor following the AIFS model."""
+        """Update time based inputs."""
 
-        # Copy prognostic field output to the input tensor
-        x = x.roll(-1, dims=1)
-        x[:, -1, :, self.prognostic_input_mask] = y[..., self.prognostic_output_mask]
+        # Select only inputs
+        x = x[..., self.model.data_indices.data.input.full]
```

Comment:
Is this something that has an API doc for or did you have to dig around anemoi  to find it?
(just wanted to see if theres a link that could be added as a comment to serve as reference.